code/datums/wounds/robotic/robotic_blunt.dm

```diff
@@ -0,0 +1,1027 @@
+	to_chat(victim, span_userdanger("[user] begins softly pressing against your collapsed [limb.plaintext_zone]!"))
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	if(!do_after(user, 8 SECONDS, target = victim, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return
+	mold_metal(user)
+	return TRUE
+
+// Once our superstructure is heated (T2 robotic burn or 125% burn damage) we can aggro grab and start pushing the metal around
+/datum/wound/blunt/robotic/critical/proc/mold_metal(mob/living/carbon/human/user)
+	var/chance = 40
+
+	if (HAS_TRAIT(user, TRAIT_KNOW_ROBO_WIRES))
+		chance *= 3
+	if (HAS_TRAIT(user, TRAIT_KNOW_ENGI_WIRES))
+		chance *= 3
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		chance *= 2
+
+	var/their_or_other = (user == victim ? "their" : "[user]'s")
+	if ((user != victim && user.combat_mode))
+		user.visible_message(span_danger("[user] molds [their_or_other] [limb.plaintext_zone] into a really silly shape! What a goofball!"))
+		limb.receive_damage(brute = 40, wound_bonus = CANT_WOUND, damage_source = user)
+	else if (prob(chance))
+		user.visible_message(span_green("[user] carefully molds [their_or_other] [limb.plaintext_zone] into the proper shape!"))
+		to_chat(victim, span_green("Your [limb.plaintext_zone] has been molded into the proper shape! Your next step is to use a screwdriver/wrench to secure your internals."))
+		set_superstructure_status(TRUE)
+	else
+		user.visible_message(span_warning("[user] screws up, damaging [their_or_other] [limb.plaintext_zone] in their efforts to help!"))
+		limb.receive_damage(brute = 5, damage_source = user, wound_bonus = CANT_WOUND)
+
+	if (HAS_TRAIT(user, TRAIT_RESISTHEAT) || HAS_TRAIT(user, TRAIT_RESISTHEATHANDS))
+		return
+
+	to_chat(user, span_warning("You burn your hand on [victim]'s [limb.plaintext_zone]!"))
+	var/obj/item/bodypart/affecting = user.get_bodypart("[(user.active_hand_index % 2 == 0) ? "r" : "l" ]_arm")
+	affecting?.receive_damage(burn = 5)
+
+// T2 burn wounds are required to mold metal, which finished the first step of treatment. Aggrograb someone and use a welder on them for a guaranteed wound with no damage
+/datum/wound/blunt/robotic/critical/proc/heat_metal(obj/item/welder, mob/living/user)
+	if (!welder.tool_use_check())
+		return TRUE
+
+	var/their_or_other = (user == victim ? "their" : "[user]'s")
+	if (user)
+		user.visible_message(span_notice("[user] carefully holds [welder] to [their_or_other] [limb.plaintext_zone], slowly heating it..."))
+
+	var/delay_mult = 1
+	if (HAS_TRAIT(src, TRAIT_WOUND_SCANNED))
+		delay_mult *= 0.75
+
+	if (!welder.use_tool(target = victim, user = user, delay = 10 SECONDS * delay_mult, volume = 50, extra_checks = CALLBACK(src, PROC_REF(still_exists))))
+		return TRUE
+
+	var/wound_path = /datum/wound/burn/robotic/overheat/severe
+	if (user != victim && user.combat_mode)
+		wound_path = /datum/wound/burn/robotic/overheat/critical
```

Comment:
Being able to immediately deal a critical level wound with no warning feels busted

Reply:
Burn wounds actually act pretty different from what you might expect

The only consistent danger from a synthetic burn wound is the fact it raises your bodytemp, and its fairly easy to treat
I'd say its possible this is a bad idea, but at the same time 1. You already _have_ a critical wound, 2. I cant really say for sure how bad of an idea it might be without first seeing how burn wounds function

Reply:
Still, I'm willing to remove this and replcae it with something more mellow